ingress/controllers/nginx/nginx.tmpl

```diff
@@ -73,6 +73,9 @@ http {
 
     client_max_body_size "{{ $cfg.bodySize }}";
 
+    # disable warning
+    uninitialized_variable_warn off;
```

Comment:
isn't this dangerous? is there  a specific variable we're expecting to be uninitialized? if so pleas doc it